build-local.sh

```diff
@@ -0,0 +1,6 @@
+echo "installing tutors package"
+cd content/
+npm install tutors@1.6.4
```

Comment:
Can we have this as a pre-requisite in the `README` file rather than having to install tutors each time we try to build it?

Reply:
yes definitely